front/src/components/login/LoginForm.js

```diff
@@ -0,0 +1,63 @@
+import React, { Component } from 'react';
+import { LoginButton, StyledCol, ForgotPassword, ErrorText} from './styledLoginForm';
+import { Form, Input, Col, Typography, Row } from 'antd';
+import 'antd/dist/antd.css';
+
+const { Title } = Typography;
+const layout = { 
+    labelCol: { span: 8, },
+    wrapperCol: { span: 16, },
+};
+
+export class LoginForm extends Component {
+
+    async validatePassword(_, value) {
+        const passwordRegex = /[a-zA-Z\d]{6,}$/;
+    
+        if (passwordRegex.test(value)) {
+          return Promise.resolve();
+        } else {
+          return Promise.reject('Password must be longer than 6 symbols');
+        }
+    }
+
+    async validateEmail(_, value) {
```

Comment:
Don't write async methods inside the component

Reply:
done